src/bridge/ui_commands.rs

```diff
@@ -148,6 +149,11 @@ impl ParallelCommand {
             ParallelCommand::FileDrop(path) => {
                 nvim.command(format!("e {path}").as_str()).await.ok();
             }
+            ParallelCommand::SetBackground(background) => {
+                nvim.command(&format!("set background={}", background))
```

Comment:
Minor nit: For the sake of consistency with the `format!` call two lines above, maybe use `.as_str()` instead.